src/sites/link/linkdrop.net.js

```diff
@@ -450,12 +450,12 @@
 
     async getURL (id) {
       while (true) {
-        const url = await $.post('getlink.php', {
-          id,
+        /*global jQuery*/
+        jQuery.post('getlink.php', {id: id}).done(function (url) {
```

Comment:
I know, dirty, but somehow $.post(... only returned the html code of the current website while jquery post worked

Reply:
`jQuery` -> `$.window.jQuery`.
I think GreaseMonkey (to be precisely, Firefox) need this to pass its sandbox rule.